nautobot/dcim/api/serializers.py

```diff
@@ -1015,7 +1015,7 @@ class Meta:
         opt_in_fields = ["computed_fields"]
 
 
-class InterfaceSerializer(
+class InterfaceSerializerVersion12(
```

Comment:
Let's add a `TODO` here to collapse this with `InterfaceSerializer` in 2.0.

Unfortunately because of the mixin needs for the newer version we can't do something like https://github.com/nautobot/nautobot/blob/develop/nautobot/ipam/api/serializers.py#L443-L444